### ISSUE TITLE
mongooseim.app.src Description

### ISSUE THREAD

**Issue by ruanpienaar** [esl/MongooseIM#484]
The proper type to use for the description in the application resource file is a string.
(http://www.erlang.org/doc/design_principles/applications.html)
Chapter 7.3

```Erlang
{application,mongooseim,
             [{description,mongooseim},
              {vsn,git},
              {modules,[]},
              {registered,[]},
              {applications,[]}]}.
```

VS

```Erlang
{application,mongooseim,
             [{description,"mongooseim"},
              {vsn,git},
              {modules,[]},
              {registered,[]},
              {applications,[]}]}.
```

We've noticed this causes unhanded crashes in wombat. Where a piece of code expected the application description & version to be a string.

**Comment by ruanpienaar** [comment]
[PR](https://github.com/esl/MongooseIM/pull/485)

**Comment by ppikula** [comment]
fixed in #485
